src/marqo/connections.py

```diff
@@ -59,7 +59,8 @@ def init_from_app(self, host, port):
             self.faulty = False
 
             t1 = time.time()
-            logger.info(f"Took {((t1-t0)*1000):.3f}ms to connect to redis and load scripts.")
+            logger.debug(f"Took {((t1-t0)*1000):.3f}ms to connected to redis...")

```

Comment:
nit: wrong grammar: "connected". You can leave the debug message as "to connect to redis and load scripts".

Reply:
Another note, it may be odd that redis is only mentioned in debug mode here once. In `run_marqo.sh`, it is referred to as throttling, even in debug mode. Maybe when in debug mode, we can always refer to it as redis.

Reply:
Fixed grammar. We aren't focusing on optimising the debug experience for now

Reply:
noted!